workflow/parse.py

```diff
@@ -0,0 +1,64 @@
+import os 
```

Comment:
Please add your authorship and License (Apache 2.0) on top, you may look it up in one of my other scripts to notice how I do it.